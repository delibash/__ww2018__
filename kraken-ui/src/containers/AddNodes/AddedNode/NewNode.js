import React from 'react';
import PropTypes from 'prop-types';
import {
  StyledItem,
  StyledItemHeader,
} from './styled';
import { Label } from './../../../components';

const NewNode = props => {
  const { nodeName } = props;
  return (
    <StyledItem>
      <StyledItemHeader>
        <Label>{ nodeName }</Label>
        <Label infoLabel>Node Added!</Label>
      </StyledItemHeader>
    </StyledItem>
  );
};

NewNode.propTypes = {
  nodeName: PropTypes.string.isRequired
};

export default NewNode;
