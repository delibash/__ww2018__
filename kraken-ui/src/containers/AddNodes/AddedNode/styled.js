import styled from 'styled-components';

const StyledContainer = styled.ul``;

const StyledItem = styled.li``;

const StyledItemHeader = styled.section`
  display: flex;
  margin: 1.8rem 0;
  justify-content: space-between;
  align-items: center;
`;

export {
  StyledContainer,
  StyledItem,
  StyledItemHeader,
};