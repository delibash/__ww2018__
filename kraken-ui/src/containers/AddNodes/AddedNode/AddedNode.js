import React from 'react';
import PropTypes from 'prop-types';
import NewNode from './NewNode';
import {
  StyledContainer,
} from './styled';

const AddedNode = props => {
  const {
    added
  } = props;
  
  return (
    <StyledContainer>
      <NewNode {...added} />
    </StyledContainer>
  );
};

AddedNode.propTypes = {
  added: PropTypes.object
};

export default AddedNode;