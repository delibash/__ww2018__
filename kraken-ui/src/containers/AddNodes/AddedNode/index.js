import AddedNode from './AddedNode';

export default AddedNode;