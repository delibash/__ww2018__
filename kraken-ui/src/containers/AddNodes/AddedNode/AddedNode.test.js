import React from 'react';
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import AddedNode from './AddedNode';

test('Renders AddedNode Container', () => {
  const tree = renderer.create(
    <AddedNode
      added={{nodeName: ''}}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});