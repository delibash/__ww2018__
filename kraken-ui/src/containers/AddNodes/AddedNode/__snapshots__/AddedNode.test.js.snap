// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders AddedNode Container 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 1.8rem 0;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  padding: .6rem 1.2rem;
  font-size: 1.3rem;
  color: rgba(100,107,255,1);
  font-weight: 600;
  border: .1rem solid rgba(0,0,0,.25);
  border-radius: 1.6rem;
  background-color: rgba(234,236,242,1);
}

.c2 {
  padding: .6rem 1.2rem;
  font-size: 1.3rem;
  color: rgba(0,0,0,1);
  font-weight: 300;
  border: 0 none;
  border-radius: .2rem;
  background-color: rgba(234,236,242,1);
}

<ul
  className=""
>
  <li
    className=""
  >
    <section
      className="c0"
    >
      <span
        className="c1"
      >
        
      </span>
      <span
        className="c2"
        color="rgba(0, 0, 0, 1)"
        fontWeight={300}
      >
        Node Added!
      </span>
    </section>
  </li>
</ul>
`;
